ne-globe.go

```diff
@@ -0,0 +1,87 @@
+package apiary
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	log "github.com/sirupsen/logrus"
```

Comment:
@hepplerj: I agree with @qtrinh2 . Unless I'm mistaken, we don't use logrus elsewhere, even though it can used as a drop in replacement. I'd recommend removing it here in favor of the standard library logger. You probably will also want to run `go mod tidy`. 

@qtrinh2: Logrus is in the `go.mod` list of dependencies, but as an indirect dependency. We do not depend on it directly.